templates/repo/issue/list.tmpl

```diff
@@ -81,15 +81,15 @@
 				{{end}}
 			{{end}}
 		</div>
-		<div id="issue-filters" class="issue-list-toolbar">
-			<div class="issue-list-toolbar-left">
+		<div id="issue-filters" class="ui stackable grid">
+			<div class="six wide column issue-list-toolbar-left">
 				{{if and ($.CanWriteIssuesOrPulls) (gt (len .Issues) 0)}}
 					<input type="checkbox" autocomplete="off" class="issue-checkbox-all gt-mr-4" title="{{.locale.Tr "repo.issues.action_check_all"}}">
 				{{end}}
 				{{template "repo/issue/openclose" .}}
 			</div>
-			<div class="issue-list-toolbar-right">
-				<div class="ui secondary filter stackable menu labels">
+			<div class="ten wide right aligned column issue-list-toolbar-right">
+				<div class="ui secondary filter stackable menu labels issues-menu">
```

Comment:
Does it needs a new "issues-menu"? It looks like ".issue-list-toolbar-right > .filter.menu" is already for it?

Reply:
Chenged.